oncogene.js

```diff
@@ -56,6 +56,10 @@ class Oncogene {
 
     setVal(key, selectedVarient) {
         const step = this.getStep()
+        if(selectedVarient.skip === true) {
```

Comment:
It depends, but even if you want check strict equality then in validation you should check that `variant.skip !== true`. User can set `variant.skip = 0` for example.